extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -59,9 +54,32 @@ sub should_send {
 }
 
 sub send {
+    my ( $self, $message ) = @_;
+
     my $logger = Bugzilla->push_ext->logger;
 
-    $logger->info('AUDIT');
+    my $data = $message->payload_decoded;
+    my $bug_data = $self->_get_bug_data($data) || return 0;
+    my $bug = Bugzilla::Bug->new( { id => $bug_data->{id}, cache => 1 } );
+
+    if ( any { $_->{name} =~ /core-security/ } @{$bug->groups()} ) {
+        $logger->info('HAS SECURITY!');
+
+        my @attachments = grep {
+            $_->isobsolete == 0 &&
+            $_->contenttype eq PHAB_CONTENT_TYPE &&
+            $_->{attacher}->{realname} eq 'phabricatorbot'
```

Comment:
Should be:

$_->attacher->login eq 'phab-bot@bmo.tld'

or whatever we set up the login to be.